web/lang/en_gb.php

```diff
@@ -76,6 +76,7 @@
     'AlarmRefImageBlendPct' => 'Alarm Reference Image Blend %ge',
     'AlarmRGBUnset'         => 'You must set an alarm RGB colour',
     'AllTokensRevoked'      => 'All Tokens Revoked',
+    'AllTags'               => 'All Tags',
```

Comment:
There is actually no problem with the lookup key having spaces, so I'd like to get away from using AllKeys and instead just use All Keys. Then we don't have to clutter up en_gb because if there isn't a translation, we just pass back the key.

Reply:
Isn't in beneficial to have all of the strings present in the main file (en_gb.php) so that it's easy to see what needs to be translated to other languages?

None of the existing keys have spaces. 

Should we keep this consistent and then update all of the keys to have spaces in the future?

Reply:
There are lots of potential translations that I didn't add because I used spaces in the key already.  It would be nice for people doing the translations to have a master list yes, but at this point, we are trying to minimise developer time and maximise performance.  So if we were going to do it, I would do it in a template file that is not generally used (because the php compiler has to go through all this stuff).  So if you want to put the effort in, go ahead.

Reply:
I agree.  Performance should be the main driver.  I'll make this change.

Reply:
Fixed with commit https://github.com/ZoneMinder/zoneminder/pull/3719/commits/145b254ace2a913b42da446d85b1b45bc28b4885